kube/src/api/typed.rs

```diff
@@ -195,10 +195,13 @@ where
 
     /// Patch a resource a subset of its properties
     ///
-    /// Note that some of the [`PatchStrategy`](super::PatchStrategy) variants require different
-    /// serialization. The original strategies such as [`Merge`], [`JSON`] and [`Strategic`] should be
-    /// serialized using [`serde_json::to_vec`].
-    /// When using [`Apply`], you **must** serialize your data using [`serde_yaml::to_vec`]:
```

Comment:
This line esp. confuses me: AFAIK yaml is superset of json, so json serialization should always be enough

Reply:
Yeah.. We're using serde_json for apply just fine.

Reply:
Ah, that line originated from me not getting the `crd_apply` example to work on 1.16 or 1.15 but it always worked with `serde_yaml`, see #176. There's still something weird about that example even on 1.19, so will revisit that, but if it works in general, then we should remove these lines as suggested.